include/psp2/motion.h

```diff
@@ -17,47 +17,192 @@ extern "C" {
 /* struct */
 
 typedef struct SceMotionState {
-	unsigned int timestamp;
-	SceFVector3 acceleration;
-	SceFVector3 angularVelocity;
-	uint8_t reserved1[12];
-	SceFQuaternion deviceQuat;
-	SceFMatrix4 rotationMatrix;
-	SceFMatrix4 nedMatrix;
-	uint32_t reserved2;
-	SceFVector3 basicOrientation;
-	SceULong64 hostTimestamp;
-	uint8_t reserved[40];
+	unsigned int timestamp;			//!< Local timestamp of the device
+	SceFVector3 acceleration;		//!< Accelerometer data
+	SceFVector3 angularVelocity;	//!< Angular velocity data
+	uint8_t reserved1[12];			//!< Reserved data
+	SceFQuaternion deviceQuat;		//!< Orientation data as a Quaternion
+	SceFMatrix4 rotationMatrix;		//!< Orientation data as a rotation matrix
+	SceFMatrix4 nedMatrix;			//!< Magnetometer orientation matrix
+	uint8_t reserved2[4];			//!< Reserved data
+	SceFVector3 basicOrientation;	//!< Orientation of the device in reference to gravity
+	SceULong64 hostTimestamp;		//!< Timestamp of the data in microseconds
+	uint8_t reserved3[36];			//!< Reserved data
+	uint8_t magFieldStrength;		//!< Either 0 or 2 depending on surrounding metal objects
```

Comment:
That's actually stability, not strength (0 being unstable, 2 - stable). And can we maybe have enum for values?

Reply:
Strength was a suggestion, I can rename and add enums once im home from work.
Im also planning on adding more doc details. Any recommendations for the enums? Testing this was a pain since it seemed to provide very inconsistent results.

Reply:
Something like
```
SCE_MOTION_MAGFIELD_UNSTABLE
SCE_MOTION_MAGFIELD_UNUSED
SCE_MOTION_MAGFIELD_STABLE
```
i guess